oci/config.py

```diff
@@ -0,0 +1,46 @@
+from collections import namedtuple
+import ConfigParser
```

Comment:
This works only on python 2, which will soon be gone. We can use this to make it work on both python 2 and 3:

    from six.moves import configparser

Reply:
Done